wandb/integration/yolov8/yolov8.py

```diff
@@ -184,6 +196,41 @@ def teardown(self, _trainer: BaseTrainer) -> None:
         self.run.finish()
         self.run = None
 
+    def on_val_start(
+        self,
+        validator: BaseValidator,
+    ) -> None:
+        """On validation start we create a wandb table to store the sample predictions
+        and patch the validator's `plot_predictions` method to plot the predictions to the wandb table .
+        """
+        class_names = sorted(validator.names.values())
```

Comment:
in the yolov8 i saw that there is also a ClassificationValidator. Is it not interesting to visualize?